src/components/backend-ai-credential-view.ts

```diff
@@ -1185,9 +1185,22 @@ export default class BackendAICredentialView extends BackendAIPage {
 
       switch (action) {
         case 'add':
+          this._showTab(
+            this.shadowRoot?.querySelector('mwc-tab[title=credential-lists]'),
+          );
+          this.shadowRoot
+            ?.querySelector('mwc-tab-bar.main-bar')
+            ?.setAttribute('activeindex', '1');
           await this._launchKeyPairDialog();
           break;
       }
+    } else if (location.search.includes('manage')) {
```

Comment:
It is awkward to handle `manage` related codes in the `_runAction()`.
Let's rename the function name or separate them.